webplugin/css/app/mck-sidebox-1.0.css

```
@@ -4389,20 +4384,24 @@ ul.mck-intent-menu > li:hover {
     font-size: 14px;
     color: #555;
     outline: none;
-    background-color: #fff;
     border: 0;
-    /* position: relative; */
-    z-index: 2;
-    /* float: left; */
-    /* display: table-cell; */
+    z-index: auto;
     border-radius: 3px;
     padding: 10px 0px;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

padding is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)